### DISCUSSION TITLE
1.8.0 release status?

### DISCUSSION THREAD

**Discussion by AMDmi3** [aio-libs/yarl#689] [Q&A]
I'm puzzled about 1.8.0 release status - is it official? There's a tag, but nither GH nor PyPI release. Should I update e.g. FreeBSD port from 1.7.2? Ping @q0w

**Comment by 93578237** [comment]
Hi, I think, ci is failed because of this https://github.com/aio-libs/create-release/pull/2, so nothing is deployed. I would suggest (as not a maintainer/member of aio-libs project) waiting for the official release.

**Comment by mjpieters** [comment]
I [raised it as a bug](https://github.com/aio-libs/yarl/issues/698) before having found this post.

I maintain a few other aiolibs projects, I'm not promising anything but I'll try to unbreak the CI here where I can.
